src/main/main.c

```diff
@@ -311,8 +311,14 @@ void init(void)
     spiInit(SPIDEV_1);
 #endif
 #ifdef USE_SPI_DEVICE_2
+#ifdef OMNIBUSF4
```

Comment:
Nothing sinister (or super secret). I am looking at two new target methods. 

preInit and postInit

This will then allow everything target specific to be removed from main.c

So if defined you can setup some IO - e.g. pull all CS high etc